test/SparqlAntlrParserTest.cpp

```diff
@@ -56,47 +56,40 @@ TEST(SparqlParser, Prefix) {
     string s = "PREFIX wd: <www.wikidata.org/>";
     ParserAndVisitor p{s};
     auto context = p.parser_.prefixDecl();
-    p.visitor_.visitPrefixDecl(context);
-    const auto& m = p.visitor_.prefixMap();
-    ASSERT_EQ(2ul, m.size());
-    ASSERT_TRUE(m.at("wd") == "<www.wikidata.org/>");
-    ASSERT_EQ(m.at(""), "<>");
+    auto prefix = p.visitor_.visitTypesafe(context);
+    ASSERT_EQ(prefix, SparqlPrefix("wd", "<www.wikidata.org/>"));
   }
   {
     string s = "wd:bimbam";
-    ParserAndVisitor p{s};
-    auto& m = p.visitor_.prefixMap();
-    m["wd"] = "<www.wikidata.org/>";
+    SparqlQleverVisitor::PrefixMap m{{"wd", "<www.wikidata.org/>"}};
+    ParserAndVisitor p{s, m};
 
     auto context = p.parser_.pnameLn();
     auto result = p.visitor_.visitTypesafe(context);
     ASSERT_EQ(result, "<www.wikidata.org/bimbam>");
```

Comment:
I think we have a one-liner for this in the `sparqlParserHelpers` which can be even shortened using the `bind_front`

Reply:
And maybe store that default `PrefixMap` if it is always the same anyway (that is code from the time where I still wrote very redundant and verbose tests, you now get to fix that:)